bookstore/_version.py

```diff
@@ -21,3 +21,23 @@
 version_info = (2, 3, 0, "", "dev")
 
 __version__ = ".".join(map(str, version_info[:3])) + ".".join(version_info[3:])
+
+
+def _check_version(bookstore_version, log):
+    """Check version and log status"""
+    if not bookstore_version:
+        log.warning(
+            f"Bookstore has no version header, which means it is likely < 2.0. Expected {__version__}"
+        )
+        return
```

Comment:
Don't think you need any of the returns if we're not returning a palyoad and all branches are gated on conditionals.